src/buildfromsource/FSharp.Compiler.Private/FSharp.Compiler.Private.fsproj

```diff
@@ -641,6 +641,25 @@
     <PackageReference Include="Microsoft.DiaSymReader.PortablePdb" Version="$(MicrosoftDiaSymReaderPortablePdbPackageVersion)" />
     <PackageReference Include="Microsoft.DiaSymReader" Version="$(MicrosoftDiaSymReaderPackageVersion)" />
     <PackageReference Include="System.ValueTuple" Version="$(SystemValueTuplePackageVersion)" />
+    <PackageReference Include="System.Collections.Immutable" Version="1.5.0-preview2-26401-03"></PackageReference>
```

Comment:
These merge conflicts need to be resolved.  The version numbers were moved to `src/buildfromsource/targets/PackageVersions.props`.

This comment holds for all changes under `src/buildfromsource`.

If you want to disable dependency uptake for this then also comment out the bottom line of that `PackageVersions.props`.